Client/src/Components/Toast/index.jsx

```diff
@@ -1,63 +1,30 @@
-import { useState, useEffect } from "react";
 import ComplexAlert from "../../Components/Icons/ComplexAlert/ComplexAlert";
-import "./index.css"; // Import your CSS file for toast styles
+import AnnouncementsDualButtonWithIcon from "../../Components/Announcements/AnnouncementsDualButtonWithIcon/AnnouncementsDualButtonWithIcon";
+import "react-toastify/dist/ReactToastify.css";
+import { toast, ToastContainer } from "react-toastify";
+import "./index.css";
 
-const Toast = () => {
-  const [showToast, setShowToast] = useState(false);
-  const [progress, setProgress] = useState(100); // Start with full progress
-
-  useEffect(() => {
-    let timer;
-    if (showToast) {
-      timer = setTimeout(() => {
-        setShowToast(false);
-      }, 5000); // 5000 milliseconds = 5 seconds
-
-      // Interval to update progress bar
-      const interval = setInterval(() => {
-        setProgress((prevProgress) => prevProgress - (100 / 5000 * 100)); // Decrease progress based on time
-      }, 100); // Update every 100 milliseconds
-
-      return () => {
-        clearTimeout(timer);
-        clearInterval(interval);
-      };
-    }
-  }, [showToast]);
-
-  const toggleToast = () => {
-    setShowToast(!showToast);
-    setProgress(100); // Reset progress when showing toast
+const ToastComponent = () => {
+  const displayMsg = () => {
+    toast(
+      <AnnouncementsDualButtonWithIcon
+        icon={<ComplexAlert theme="red" />}
+        subject="There was a problem with that action"
+        body="Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid pariatur, ipsum dolor."
+        esc="Dismiss"
+        primary="Learn more"
+        dismiss={() => toast.dismiss()}
```

Comment:
Dismissing toasts doesn't seem to work at the moment, am I clicking on the wrong thing maybe? :thinking:

Reply:
Oh no! Really? It was working for me before. Maybe I changed something the last second. Let me take a look and address this issue.